### PR TITLE
refactor: edit mutation to update title of welcome post

### PR DESCRIPTION
**Pull request by sshanzel**
This is to allow the welcome post's title to be updated as part of v6.

### CODE REVIEW

File: src/schema/posts.ts [hunk 1]
```diff
@@ -1028,23 +1028,23 @@ export const resolvers: IResolvers<any, Context> = {
 
         if (content.length > MAX_CONTENT_LENGTH) {
           throw new ValidationError(
-            'Content has a maximum length of 80 characters',
+            'Content has a maximum length of 4000 characters',
```
Comment (rebelchris) [inline src/schema/posts.ts:1031]: Title also has a max length perhaps we should also check that here?

**Review by rebelchris — CHANGES_REQUESTED** [review]
Just one suggestion otherwise looks amazing

File: src/schema/posts.ts (hunk 1 above)
Comment (sshanzel) [reply src/schema/posts.ts:1031]: There is, we are already checking it from 1024. I just fixed the typo here.

**Review by rebelchris — APPROVED** [review]
Looks great

**Review by idoshamun — APPROVED** [review]
Awesome!

**Pull request merged**